### PR TITLE
new-project: dynamically insert Dagster and Dagit versions

### PR DESCRIPTION
**Pull request by taljaards**
## Summary
`dagster new-project PROJECT_NAME` always generated a fixed requirement for 0.10.2.
I changed it to use the current version of Dagster (that was used to run the command) instead of the fixed version.

## Checklist

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: python_modules/dagster/dagster/generate/generate.py [hunk 1]
```diff
@@ -59,7 +61,13 @@ def generate_new_project(path: str):
                 # Jinja template names must use the POSIX path separator "/".
                 template_name = src_relative_file_path.replace(os.sep, posixpath.sep)
                 template = env.get_template(name=template_name)
-                f.write(template.render(repo_name=repo_name))
+                f.write(
+                    template.render(
+                        repo_name=repo_name,
+                        dagster_version=dagster_version,
+                        dagit_version=dagit_version,
```
Comment (rexledesma) [inline python_modules/dagster/dagster/generate/generate.py:68]: cc @catherinewu, @sryza thoughts on enforcing dagit and dagster version to be the same here?
```suggestion
                        dagit_version=dagster_version,
```

File: python_modules/dagster/dagster/generate/generate.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 import posixpath
 
 import jinja2
+from dagit.version import __version__ as dagit_version
```
[the remark this answers is not in the document]
Comment (taljaards) [reply python_modules/dagster/dagster/generate/generate.py:5]: I actually wanted to do so.
I'm just installing a proper dev environment in WSL2 so I can run tests locally before submitting a fix.

File: python_modules/dagster/dagster/generate/generate.py (hunk 1 above)
Comment (helloworld) [reply python_modules/dagster/dagster/generate/generate.py:68]: For now, we need both versions to be the same (we don't explicitly support running different versions of Dagster and Dagit with each other), so yes we should do this.

**Comment by rexledesma** [comment]
Thanks!

**Pull request merged**